utils/setStatus.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+const statusCodes = require('http').STATUS_CODES
+    , setStatus = () => {
+        return function (status) {
+            /* eslint-disable no-invalid-this */
+            const that = this;
+            /* eslint-enable no-invalid-this */
+            let k;
+
+            if (status in statusCodes) {
```

Comment:
would be more in line with the existing code style to either use `typeof statusCodes[status] !== 'undefined'` or use https://github.com/ansble/monument/blob/master/utils/tools.js which has an `isUndefined` check and negating it. You'll get a lint warning but that is ok.